README.md

```diff
@@ -67,6 +67,25 @@ local IsVisible = RegionState:IsRegionVisible("Region1") --Bool for if a region
 local InRegion = RegionState:IsInRegion("Region1", Vector3.new()) --Bool for if a point is in a region
 ```
 
+## Quicker Region Setup
```

Comment:
I wouldn't have this as the same type of subsection as `RegionState` and `ModelCulling` when it should be part of `RegionState` or under it.